### ISSUE TITLE
Unable to use version 2.10.0 of the SDK

### ISSUE THREAD

**Issue by davidkhala** [oracle/oci-typescript-sdk#115]
Scenario
In Dec 20, 2021, I try to bump my npm dependency to latest
```
"dependencies": {
    "oci-sdk": "latest"
  },
```
I see error like 

```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for oci-appmgmtcontrol@2.10.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

```

**Comment by jodoglevy** [comment]
Thanks for reporting, we'll take a look

**Comment by y-chandra** [comment]
There was an issue while publishing the latest version of the SDK and one of the packages is missing in the latest version (`2.10.0`). Please use `2.9.1` version of the SDK.

**Comment by y-chandra** [comment]
@davidkhala - Can you please try again with the latest version? We've fixed this issue and published version `2.10.0` of the package `oci-appmgmtcontrol`

**Comment by davidkhala** [comment]
@y-chandra Resolved.
